test/fixtures/simple_instance/.gitignore

```diff
@@ -0,0 +1,5 @@
+config.sh
```

Comment:
@adrienthebo In general I agree, but given that the `examples/` directory is generated from copying these files over, I'd like to leave some basic terraform-related files `.gitignored` a second time here (e.g. `.terraform` -- given that it's a good practice for any terraform codebase to `gitignore` this directory).

I'll remove `config.sh` and do some consolidation otherwise.